### PR TITLE
Add DeleteRange API

### PR DESCRIPTION
**Pull request by birdstorm**
Signed-off-by: birdstorm <samuelwyf@hotmail.com>

### CODE REVIEW

File: src/main/java/org/tikv/common/KVClient.java [hunk 1]
```diff
@@ -141,7 +141,7 @@ public ByteString get(ByteString key, long version) throws GrpcException {
           () -> doSendBatchGetInBatchesWithRetry(singleBatchBackOffer, batch, version));
     }
 
-    return getKvPairs(completionService, batches);
+    return getKvPairs(completionService, batches, BackOffer.BATCH_GET_MAX_BACKOFF);
```
Comment (marsishandsome) [inline src/main/java/org/tikv/common/KVClient.java:144]: it's not related to the pr title.

File: src/main/java/org/tikv/common/region/RegionStoreClient.java [hunk 2]
```diff
@@ -846,7 +846,7 @@ private void rawDeleteHelper(RawDeleteResponse resp, TiRegion region) {
       throw new TiClientInternalException("RawDeleteResponse failed without a cause");
     }
     String error = resp.getError();
-    if (error != null && !error.isEmpty()) {
+    if (!error.isEmpty()) {
```
Comment (marsishandsome) [inline src/main/java/org/tikv/common/region/RegionStoreClient.java:849]: it's not related to the pr title.

File: src/main/java/org/tikv/common/region/RegionStoreClient.java [hunk 3]
```diff
@@ -879,7 +879,7 @@ private void rawPutHelper(RawPutResponse resp) {
       throw new TiClientInternalException("RawPutResponse failed without a cause");
     }
     String error = resp.getError();
-    if (error != null && !error.isEmpty()) {
+    if (!error.isEmpty()) {
```
Comment (marsishandsome) [inline src/main/java/org/tikv/common/region/RegionStoreClient.java:882]: it's not related to the pr title.

File: src/main/java/org/tikv/raw/RawKVClient.java [hunk 4]
```diff
@@ -15,7 +15,7 @@
 
 package org.tikv.raw;
 
-import static org.tikv.common.util.ClientUtils.getKvPairs;
+import static org.tikv.common.util.ClientUtils.*;
```
Comment (marsishandsome) [inline src/main/java/org/tikv/raw/RawKVClient.java:18]: run `mvn com.coveo:fmt-maven-plugin:format`

File: src/main/java/org/tikv/common/region/RegionStoreClient.java [hunk 5]
```diff
@@ -1003,6 +1003,40 @@ private void handleRawBatchPut(RawBatchPutResponse resp) {
     return resp.getKvsList();
   }
 
+  public void rawDeleteRange(BackOffer backOffer, ByteString startKey, ByteString endKey) {
```
Comment (marsishandsome) [inline src/main/java/org/tikv/common/region/RegionStoreClient.java:1006]: maybe add some notes, the caller may want to know
- [startKey, endKey] or
- [startKey, endKey) or
- (startKey, endKey] or
- (startKey, endKey)

**Review by marsishandsome — APPROVED** [review]
others LGTM

File: src/main/java/org/tikv/raw/RawKVClient.java (hunk 4 above)
Comment (marsishandsome) [reply src/main/java/org/tikv/raw/RawKVClient.java:18]: AvoidStarImport

File: src/main/java/org/tikv/common/region/RegionStoreClient.java (hunk 3 above)
Comment (birdstorm) [reply src/main/java/org/tikv/common/region/RegionStoreClient.java:882]: They are related to similar changes in `rawDeleteRangeHelper`

File: src/main/java/org/tikv/common/region/RegionStoreClient.java (hunk 2 above)
Comment (birdstorm) [reply src/main/java/org/tikv/common/region/RegionStoreClient.java:849]: They are related to similar changes in `rawDeleteRangeHelper`

File: src/main/java/org/tikv/common/KVClient.java (hunk 1 above)
Comment (birdstorm) [reply src/main/java/org/tikv/common/KVClient.java:144]: It is added because the corresponding API in `ClientUtils` has changed.

Comment (birdstorm) [reply src/main/java/org/tikv/common/KVClient.java:144]: ... which corresponds to the new-added `ClientUtils.getTasks()` API

File: src/main/java/org/tikv/common/region/RegionStoreClient.java (hunk 5 above)
Comment (birdstorm) [reply src/main/java/org/tikv/common/region/RegionStoreClient.java:1013]: fixed

**Review by marsishandsome — APPROVED** [review]
LGTM

**Pull request merged**